onnxruntime/core/providers/cpu/nn/pool.cc

```diff
@@ -440,19 +450,28 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     MaxPool,
     8, 9,
     KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()).TypeConstraint("I", DataTypeImpl::GetTensorType<int64_t>()),
-    Pool<float, MaxPool<8 /*VERSION*/>>);
+    PoolVersion12);
 
```

Comment:
Confusing for a class called PoolVersion12 to be handling old versions. Is there a better name?

Reply:
This will be fixed by merging Changming changes.

---
In reply to: [400575412](https://github.com/microsoft/onnxruntime/pull/3341#discussion_r400575412) [](ancestors = 400575412)